src/bp-templates/bp-nouveau/js/buddypress-search.js

```diff
@@ -10,6 +10,13 @@ jQuery(document).ready(function($) {
 			$(BP_SEARCH.autocomplete_selector).each(function() {
 				var $form = $(this),
 					$search_field = $form.find('input[name="s"], input[type=search]');
+				/**
+				 * Continue the loop if it is woocommerce product search widget
+				 */
+				if ( $( 'input[name="post_type"]', $form ).length > 0 && 'product' === $( 'input[name="post_type"]', $form ).val() ) {
```

Comment:
@mahedihasannoman Look like this is not the correct solution. We should avoid adding a such condition for post type in js.
we should find some alternate solution for this.